packages/augur-tools/src/constants.ts

```diff
@@ -13,9 +13,44 @@ export const ACCOUNTS: Account[] = [
     publicKey: "0x913dA4198E6bE1D5f5E4a40D0667f70C0B5430Eb",
```

Comment:
IMO the first account should have a comment indicating that it's considered to be the contract owner.